examples/dev-server-fargate/variables.tf

```diff
@@ -14,3 +14,10 @@ variable "lb_ingress_ip" {
   description = "Your IP. This is used in the load balancer security groups to ensure only you can access the Consul UI and example application."
   type        = string
 }
+
+variable "gossip_encryption_key" {
+  description = "Consul gossip encryption key. Must match among all Consul agents. Generated if not provided."
+  type        = string
+  default     = ""
+  sensitive   = true
+}
```

Comment:
We didn't keep the option to enable TLS for this `dev-server-fargate` example (instead, TLS is part of the acceptance tests). Do we want to enable gossip encryption in the example?

Reply:
I think we should keep the example not secure i.e. no TLS, no ACLs and no gossip encryption. Since it's meant for somebody to try it, it makes to keep it simple.